### DISCUSSION TITLE
Reports for advanced block

### DISCUSSION THREAD

**Discussion by Andr198** [YetiForceCompany/YetiForceCRM#16619] [How to]
Нow to create a report from the Sales Orders module.
Need a report on Assigned To.  What goods were sold to the person in charge.

**Comment by rskrzypczak** [comment]
Try to use widget:
- widget from filter
- chart from filter

![image](https://user-images.githubusercontent.com/10221999/199659182-0b975582-f320-4067-8cf6-5d555b871870.png)
